hack/dump/README.md

```diff
@@ -0,0 +1,140 @@
+# Aggregated Logging Utilities
+
+ ## logging-dump
+`logging-dump.sh` is a tool to gather as much information as possible from your logging cluster. In order to run the script, the user must be logged in and have a administrative permissions on project `logging`
+
+Usage:
+
+`$ oc login -u admin https://openshift.example.com:8443`
```

Comment:
to be clear, though, it looks like this would work just as well with `oc login -u system:admin` (where you don't have a token).